db/install.xml

```diff
@@ -52,6 +52,7 @@
         <FIELD NAME="includeqrcode" TYPE="int" LENGTH="1" NOTNULL="true" DEFAULT="0" SEQUENCE="false" COMMENT="Include a QR code image when displaying the password"/>
         <FIELD NAME="rotateqrcode" TYPE="int" LENGTH="1" NOTNULL="true" DEFAULT="0" SEQUENCE="false"/>
         <FIELD NAME="rotateqrcodesecret" TYPE="char" LENGTH="10" NOTNULL="false" SEQUENCE="false"/>
+        <FIELD NAME="automarkcmid" TYPE="int" LENGTH="10" DEFAULT="0" SEQUENCE="false"/>
```

Comment:
this looks suspiciously like you edited in manually - it's missing something that xmldbeditor would have added....